packages/docusaurus/src/client/templates/index.html.template.ejs

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
btw was wondering, is this file still useful?

wasn't it you that migrated from EJS to ETA?

Reply:
Haha yes it was. Actually, this file is used by https://github.com/jantimon/html-webpack-plugin, which by default uses a template engine that they call EJS but is really just lodash.template :man_shrugging: 

Maybe I should request to migrate that to Eta too :grinning: